src/main/java/org/codehaus/mojo/versions/api/AbstractVersionDetails.java

```diff
@@ -64,6 +65,7 @@ protected AbstractVersionDetails()
     {
     }
 
+    @Deprecated
```

Comment:
Please give reason, what is replacement for it, and so on

Reply:
Removed the deprecation instead. Originally added because I'd noticed that the constructor was not being used anymore, but it might come useful for testing.